airbyte-integrations/connectors/destination-pinecone/integration_tests/pinecone_integration_test.py

```diff
@@ -32,6 +33,7 @@ def _wait(self):
         print()  # Move to the next line after the loop
     
     def setUp(self):
+        os.environ["INTEGRATION_TEST"] = "True"
```

Comment:
```suggestion
```

I think we can use the built-in `PYTEST_CURRENT_TEST` env var.

 https://stackoverflow.com/a/58866220/4298208